Add .urcap installation tutorial (#6)
UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCap: doc/install_urcap.md

> # How to install the RS485 URCap

Reviewer: @urrsk Is there something like an "official" guide to install a URCap? It might make more sense to link to one instead of creating on in each repository that we create (see [ros_as_a_service](https://github.com/UniversalRobots/Universal_Robots_ROS_as_a_Service_URCap/blob/master/doc/installation.md), [ur_robot_driver (external_control)](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/doc/install_urcap_e_series.md))

Reviewer 2: It is described in the robot manual!

Reviewer: but this is unfortunately not really referencable from an online documentation, right? Sure, we can point to UR's download page, which requires a login, let the user search for the manual and then the respective chapter.

But linking to a specific online documentation directly, would be much more beneficial in my opinion.

Reviewer 3: Yes, fully agree with @fmauch.

UR support being put behind a login was a rather unwelcome surprise .. (well, some of it anyway)

Reviewer 2: You can now read the manual online on http://help.universal-robots.com/ without the need to sign in

This is a short step-by-step guide how to install the RS485 URCap in Polyscope.

> We assume that you copied the latest *RS485 URCap* from [here](https://github.com/UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCap/releases) to the `programs` folder of your e-series robot.

Reviewer: Linking back to the description how to copy it there might be useful.

Author: @fmauch Actually, that's about it with hints on how to copy. The other part in the top-level README.md just says *via scp or via usb stick*. Link back anyways?

Reviewer: Yes, that's what I meant.
```suggestion
We assume that you [copied](../README.md#installation) the latest *RS485 URCap* from [here](https://github.com/UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCap/releases) to the `programs` folder of your e-series robot.
```

## Add the URCap to the system

> After startup of the robot, click on the `hamburger` menue at the top right of

Reviewer: ```suggestion
After startup of the robot, click on the `hamburger` menu item at the top right of
```

the screen and select `Settings`.

![here](./resources/install_1.png)

> In the menue on the left, go down to `System` -> `URCaps` and click the `+` button to add a URCap to the system.

Reviewer: ```suggestion
In the menu on the left, go down to `System` -> `URCaps` and click the `+` button to add a URCap to the system.
```
